ci-operator/step-registry/hypershift/agent/create/agent/hypershift-agent-create-agent-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ create agent command ************"
```

Comment:
Seems like the commands in the `deploy_hypershift.sh` script aren't creating agents instead it's consuming agents as hypershift worker nodes. consider renaming the script and the file name e.g.  `hypershift-agent-add-worker-commands.sh`

Reply:
In `deploy_hypershift.sh` we create BMH through an, Agent to show up after a while, it is better to name it as add-worker, thanks